apps/definition_dictionary/test/dictionary_test.exs

```diff
@@ -18,45 +18,46 @@ defmodule DictionaryTest do
       assert Dictionary.Type.String.new!(name: "name") == Dictionary.get_field(dictionary, "name")
     end
 
-    data_test "get_type returns all fields with that type" do
-      dictionary =
-        Dictionary.from_list([
-          Dictionary.Type.String.new!(name: "name"),
-          Dictionary.Type.Integer.new!(name: "age"),
-          Dictionary.Type.Date.new!(name: "birthdate", format: "%Y-%m-%d"),
-          Dictionary.Type.String.new!(name: "nickname"),
-          Dictionary.Type.Map.new!(
-            name: "spouse",
-            dictionary: [
-              Dictionary.Type.String.new!(name: "name"),
-              Dictionary.Type.Wkt.Point.new!(name: "location")
-            ]
-          ),
-          Dictionary.Type.List.new!(
-            name: "friends",
-            item_type:
-              Dictionary.Type.Map.new!(
-                name: "in_list",
-                dictionary: [
-                  Dictionary.Type.String.new!(name: "name"),
-                  Dictionary.Type.Map.new!(
-                    name: "work",
-                    dictionary: [
-                      Dictionary.Type.Wkt.Point.new!(name: "location")
-                    ]
-                  )
-                ]
-              )
-          ),
-          Dictionary.Type.List.new!(
-            name: "colors",
-            item_type: Dictionary.Type.String.new!(name: "in_list")
-          )
-        ])
+    data_test "get_by_type returns all fields with that type" do
+      dictionary = [
+        Dictionary.Type.String.new!(name: "name"),
+        Dictionary.Type.Integer.new!(name: "age"),
+        Dictionary.Type.Date.new!(name: "birthdate", format: "%Y-%m-%d"),
```

Comment:
Is it worth throwing a test in here for Timestamp since that was what was giving me problems?

Reply:
I don't think so.